backend_readme.md

```diff
@@ -0,0 +1,62 @@
+# **Productivity Cube - backend**
+
+## Description
+This is a backend part of Time Management Supporting System. The whole idea is about helping user with making decisions on what he should put his focus at the moment. We create a small cube, which analyzes our day at work and then predict if the action taken during some part of the day would be efficient or will be interrupted by other factors.
+
+
+## Installing
+
+
+* Download or clone project from [this](https://github.com/Productivity-Cube/cube-backend)
+* Type in bash
+```
+npm i
+ ```
+* Then type
+```
+npm run start-dev
+```
+
+## Documentation
+* Node.Js and NPM - You can download Node.Js and NPM from https://nodejs.org/en/download/
+* [TypeScript](https://www.typescriptlang.org) - The web framework used
+
+## Usage or Screenshots
+1. Install and start *Productivity-Cube/cube-frontend*
+
+
+*Tip, need to delete later* <br>
+*should describe a general flow of using the app - how to login, which API endpoint is used there, why you don’t need a password, how to report an event etc.*
+
+## How To
+
+## FAQ
+1. NPM shows:
+```
+npm ERR! code ELIFECYCLE                                                                                  
+npm ERR! errno 1                                                                                          
+npm ERR! cube-backend@1.0.0 start-dev: `ts-node ./src/index.ts`                                           
+npm ERR! Exit status 1                                                                                    
+npm ERR!                                                                                                  
+npm ERR! Failed at the cube-backend@1.0.0 start-dev script.                                               
+npm ERR! This is probably not a problem with npm. There is likely additional logging output above.        
+```
+* You should copy .env.example to .env.

```

Comment:
This is a part of the installation guide and should be put before `npm i`,not in the FAQ section